docs/pipelines/index.md

```diff
@@ -88,6 +88,18 @@ All scripts inside pipeline folder have the following parameters.
 - **--run_after**: Can be used in conjunction with `--expname` to chain jobs to
   run one after another (only applicable on slurm). E.g. run training job with
   `--expname my-training-run` and then launch an eval with `--run_after my-training-run`.
+- **--mount_paths**: Can be used to mount additional paths to the cluster config dynamically. 
+  This is useful if you want to access some data that is not mounted in cluster config. E.g. use
+  `--mount_paths /my/remote/workspace:/workspace` to mount `/workspace` folder from the host
+  machine to the slurm job.
+- **--check_mounted_paths**: This flag offers a few different capabilities for convenience:
```

Comment:
this description is a little confusing to me. E.g. sometimes it will raise an error if the path doesn't exist, but sometimes it will create a folder for me. How do I know when it's one or the other?

Reply:
Will fix in a second pr for nits. That wont need GPU exec hopefully.

Reply:
The nain thing is `--check_mount_paths` is the flag that enables remote file and dir checks. When not set, we do just local checks of whether the path "looks correct" and mounted. When this flag is set, we confirm with remote calls to the cluster, that indeed the paths are in correct location or not. 

This is set to false by default because it is fine for one off scripts but in a complex chain of scripts - you don't want to assert on existence of data when a previous pipeline stage is going to generate that data file.